cli/deployment/config.go

```diff
@@ -236,7 +236,7 @@ func newConfig() *codersdk.DeploymentConfig {
 
 		Telemetry: &codersdk.TelemetryConfig{
 			Enable: &codersdk.DeploymentConfigField[bool]{
-				Name:    "Telemetry Enable",
+				Name:    "Telemetry",
```

Comment:
Ah, looks like this change wasn't sufficient to actually revert this rename. The new structure probably won't allow this to be named just "TELEMETRY` as an env var.